packages/rum/src/domain/rumEventsCollection/view/viewCollection.ts

```diff
@@ -0,0 +1,80 @@
+import { Configuration, getTimestamp, msToNs } from '@datadog/browser-core'
+import { RumEventCategory, RumViewEvent } from '../../../types'
+import { RumEventType, RumViewEventV2 } from '../../../typesV2'
+import { LifeCycle, LifeCycleEventType } from '../../lifeCycle'
+import { trackViews, View } from './trackViews'
+
+export function startViewCollection(lifeCycle: LifeCycle, configuration: Configuration, location: Location) {
```

Comment:
I think the `location` argument is always `window.location` now. We need to keep the `location` argument on `trackViews`, but we could remove this one.

Reply:
it is still used by some [rum.ts spec](https://github.com/DataDog/browser-sdk/blob/master/packages/rum/src/boot/rum.spec.ts#L273)

Reply:
Ah, right!